services/src/main/java/io/druid/cli/CoordinatorJettyServerInitializer.java

```diff
@@ -51,6 +51,7 @@
   public void initialize(Server server, Injector injector)
   {
     final ServletContextHandler root = new ServletContextHandler(ServletContextHandler.SESSIONS);
+    root.setInitParameter("org.eclipse.jetty.servlet.Default.dirAllowed", "false");
```

Comment:
@drcrallen redirectWelcome doesn't have to be set, because index.html is the jetty default
@nishantmonu51 we can remove dirAllowed by default, but it only matters for nodes which have actual html resource files, which is only the case for coordinator / overlord.